src/TagsInput/TagsInput.js

```diff
@@ -11,156 +16,159 @@ import InputField from './components/InputField';
 import { StyleSheet } from '../PlatformStyleSheet';
 import { Text } from '../Text';
 
-import { INPUT_MIN_WIDTH, TAGS_MIN_WIDTH } from './constants';
-
 type Props = {|
-  +label: string,
+  +fontSize: number,
   +onChangeText: (value: string) => void,
-  +placeholder: string,
-  +selected?: string[],
+  +tags: string[],
   +disabled?: boolean,
-  +fontSize: number,
+  +label?: string,
   +onClearPress?: () => void,
+  +onKeyPress?: (e: Event) => void,
+  +placeholder?: string,
   +value?: string,
-  +autoFocus: boolean,
 |};
 
 type State = {
   value: string,
   isFocus: boolean,
-  containerWidth: ?number,
-  inputWidth: ?number,
+};
+
+type Event = { nativeEvent: { key: string } };
+
+const boxShadow = isFocus => {
+  if (Platform.OS === 'web') {
+    return {
+      boxShadow: isFocus
+        ? `${defaultTokens.borderColorInputFocus} 0 0 0 2px inset`
+        : `${defaultTokens.borderColorInput} 0 0 0 1px inset`,
+    };
+  }
+  return null;
 };
 
 export default class TagInput extends React.Component<Props, State> {
-  inputRef: ?{ current: null | React.Element<typeof TextInput> };
+  inputRef: any;
+  scrollRef: any;
+
   static defaultProps = {
     fontSize: 16,
```

Comment:
Maybe we could use design tokens here, probably `fontSizeButtonLarge`. It's some kind of button, right?

Reply:
done